src/pages/Question.tsx

```diff
@@ -40,79 +38,81 @@ const MOVIES = gql`
   }
 `
 
+const GET_SCORE = gql`
```

Comment:
Moved everything into this file and deleted the "service", as that caused issues.

Reply:
Makes sense to not create a new Apollo Client. I think you solution with the lazyQuery is nice (below).